app/docs/dev/policies/meshtrafficpermission.md

```diff
@@ -0,0 +1,280 @@
+---
+title: MeshTrafficPermission (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficPermission](../traffic-permissions.md).
```

Comment:
This seems 
```suggestion
it should not be mixed with [TrafficPermission](../traffic-permissions).
```
I think now you shouldn't put the extension that's why checklink is broken

https://deploy-preview-1132--kuma.netlify.app/docs/dev/policies/meshtrafficpermission/
	404	https://deploy-preview-1132--kuma.netlify.app/docs/dev/policies/traffic-permissions.md